### PR TITLE
fix(select): add ellipsis to items counter

### PR DESCRIPTION
**Pull request by GoncaloCamaz**

### CODE REVIEW

File: packages/ui-components/src/components/select-multi-options/select-multi-options.tsx [hunk 1]
```diff
@@ -340,7 +341,9 @@ export class KvSelectMultiOptions implements ISelectMultiOptionsConfig, ISelectM
 				<slot name="select-header-label" slot="select-header-label" />
 				{this.counter && (
 					<div slot="select-header-label">
-						<div class="selected-items-label">Selected: {`${selectedOptionsLength}/${optionsLength}`}</div>
+						<kv-tooltip text={selectedItemsCountText}>
```
Comment (djcouto) [inline packages/ui-components/src/components/select-multi-options/select-multi-options.tsx:344]: We should use the tooltip truncate prop here instead.

**Review by djcouto — CHANGES_REQUESTED** [review]

File: packages/ui-components/src/components/select-multi-options/select-multi-options.scss [hunk 2]
```diff
@@ -21,7 +21,9 @@ kv-select {
 }
 
 .selected-items-label {
+	@include ellipsis();
```
Comment (djcouto) [inline packages/ui-components/src/components/select-multi-options/select-multi-options.scss:24]: I don't think we still need this after your latest changes.

**Review by djcouto — APPROVED** [review]

**Review by miguelpinto98 — APPROVED** [review]
Approved, but with one question. 
Does it really make sense to truncate/hide the selected items?

**Comment by GoncaloCamaz** [comment]
> Approved, but with one question. Does it really make sense to truncate/hide the selected items?

The dropdown width is still configured from the outside in case we want to prevent this problem.

**Pull request merged**